sema/commons/yml/loader.py

```
@@ -0,0 +1,42 @@
+from yaml.nodes import ScalarNode
+from logging import getLogger, Logger
+
+
+log: Logger = getLogger(__name__)
+
+
+class LoaderBuilder():
+    def __init__(self):
+        # this to guarantee that no add_constuctor() chain are shared
+        class ClonedLoader(SafeLoader):
+            def __new__(cls, *args, **kwargs):
```

Ruff (0.8.2)

Missing return type annotation for special method `__new__`

(ANN204)